pkg/ocm/ocm.go

```diff
@@ -310,6 +310,21 @@ func (c *SdkClient) IsAccessProtected(cluster *cmv1.Cluster) (bool, error) {
 	return enabled, nil
 }
 
+func CheckIfUserBanned(ocmClient Client, cluster *cmv1.Cluster) (bool, string, error) {
+	user, err := GetCreatorFromCluster(ocmClient.GetConnection(), cluster)
+	if err != nil {
+		return false, "encountered an issue when checking if the cluster owner is banned. Please investigate.", err
+	}
+
+	if user.Banned() {
+		noteMessage := fmt.Sprintf("User is banned %s. Ban description %s.\n Please open a proactive case, so that MCS can resolve the ban or organize a ownership transfer.", user.BanCode(), user.BanDescription())
+		logging.Warnf(noteMessage)
+		return true, noteMessage, nil
+	} else {
```

Comment:
minor nit: we could omit the `else` here :slightly_smiling_face: Just highlighting this because some linters are picky about that